kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/ArrayColumnReader.java

```diff
@@ -52,15 +55,38 @@ public ColumnVector getDataColumnVector(int batchSize) {
         return arrayVector;
     }
 
+    /**
```

Comment:
[same code](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala#L346) in Spark Parquet reader.